MusicTaste/MusicTaste/MatchingAlgorithm.m

```diff
@@ -21,16 +22,13 @@ - (void) lookForMatches{
     PFObject *music = [PFObject objectWithClassName:@"Music"];
     PFQuery *query = [PFQuery queryWithClassName:@"Music"];
     PFUser *current = [PFUser currentUser];
+    self.author = [SpotifyAPIManager shared].author;
     //Goes through Query
     [query findObjectsInBackgroundWithBlock:^(NSArray *spotifyData, NSError *error) {
-        for (PFObject *user in spotifyData){
```

Comment:
Doesn't have to go through all user to compare Spotify information now just gets matches for one user